app/views/koi/admin_crud/_form_field_composable.html.erb

```diff
@@ -1,14 +1,6 @@
-<%-
-  value = f.object.send(attr)
-  # fix for data not saving in JSON format
-  if value.is_a?(String)
-    value = JSON.parse(value)
-  end
--%>
-
 <%= react_component("KoiComposable", props: {
-  composition: value,
-  config: f.object.class.composable_crud_config,
+  composition: { data: f.object.send(attr) },
```

Comment:
Might have wanted to use public send here (which wouldn't be consistent with a lot of the codebase but I think it's usually your preference?)